modules/rhcos-install-iscsi-manual.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-manual_{context}"]
+= Installing {op-system} manually on an iSCSI boot device
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target that you want to install {op-system} on.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
```

Comment:
```suggestion
    --portal 192.168.0.10 \ <1>
```
Is this IP address supposed to be the same as the user-replaceable value here? https://github.com/openshift/openshift-docs/pull/77275/files#diff-777a0ba1b9d7fa92241577268ef5daa820fd145835e8342f7baca4834bea226fR34

If yes, I would make this the same in both places (probably `<IP_address>` with a footnote). I assume the valid value for the portal is a single IP? Do we need to specify?

Reply:
yeah it's the same, good catch! and no